app/helpers/application_helper.rb

```diff
@@ -15,7 +15,11 @@ def default_title_content
   end
 
   def active_class(name)
-    name.include?(controller_path) ? "active" : controller_path
+    if name.find { |n| request.path.ends_with?(n) }
```

Comment:
I feel like this might mess us up since we're doing string checks. Can we not change this to check the actual `controller` and `action` from the parameters?